common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -139,6 +139,14 @@ def _translate(cls, key):
                          # Used for storing xml attributes between import and export, for roundtrips
                          'xml_attributes')
 
+    # This is a categories to fields map that contains the block category specific fields which should not be
+    # cleaned and/or override while adding xml to node.
+    metadata_to_not_to_clean = {
+        # A category `video` having `sub` and `transcripts` fields
+        # which should not be cleaned/override in an xml object.
+        'video': ('sub', 'transcripts')
+    }
+
```

Comment:
Can you verify that if we remove all changes in this file then video module tests will fail?